[AI] Fixes and updates for skills (#191)
open-s4c/csb: .agents/skills/csb/SKILL.md

cat /proc/sys/kernel/perf_event_paranoid
```

Warn user if the benchmark configuration file requests both `perf` and `perf_lock` monitors, as those two do not work together. Ask the user to choose one of them in this case.

> It is known that `perf` and `perf_lock` monitors can conflict with each other, causing corruption of the `perf` monitor results. If both monitors are enabled in the configuration, ask the user to leave only one of these monitors enabled.

Reviewer: ```suggestion
In CSB,  `perf` and `perf_lock` monitors can conflict with each other, causing corruption of the `perf` monitor results. If both monitors are enabled in the configuration, ask the user to leave only one of these monitors enabled.
```
